.dprintrc.json

```diff
@@ -10,7 +10,11 @@
   "markdown": {
     "textWrap": "always"
   },
-  "includes": ["**/*.{ts,tsx,js,jsx,json,md}"],
+  "yapf": {
+    "based_on_style": "pep8",
+    "indentWidth": 4
```

Comment:
The `indentWidth: 2` above causes this plugin to use an indentation of 2, but the code uses `4` for python.